### DISCUSSION TITLE
Skiovox 124?

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#1069] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
If you could check it for me to see if it works is it at https://discord.com/channels/419123358698045453/1238543396390309938 So there is a vid at the upper section of the kajij so take a look @AshtonDavies may see it and take a look? (Btw I am already on 120 so I don't need this)

**Comment by LieutenantGst** [comment]
I bet its not a thing because its a really recent update but maybe their is

**Comment by FunnyCorgi** [comment]
The discord is not working. Can you fix this so I can see this new method(just interested)?

**Reply by AshtonDavies** [reply, marked as answer]
This is from [SKIOVOX 124 kajig](https://discord.com/channels/419123358698045453/1238543396390309938/1238543396390309938).

# Skiovox on v124
**THIS MAY NOT WORK FOR SOME PEOPLE DUE TO THE PATCH ‼️**
prereqs:
don't be stupid and have fast fingers
sign in as existing user on network not available page

# Steps:

1. Sign out and turn off your wifi
2. Open kiosk app of choice and wait for the network page to show
3. DO NEXT STEPS AS FAST AS POSSIBLE
4. Click add network and quickly type in random letters and hit enter
5. Quickly press diagnose, wifi, open in settings, and then press Close at the bottom right to exit out back to the wifi page
6. As fast as you can press log in as existing and sign in
7. Close the school window in the top right and there should be an unblocked page behind it

If all went well you should have an unblocked SKIOVOX container.

**Reply by nathblade16** [reply]
> This is from [SKIOVOX 124 kajig](https://discord.com/channels/419123358698045453/1238543396390309938/1238543396390309938).
> 
> # Skiovox on v124
> **THIS MAY NOT WORK FOR SOME PEOPLE DUE TO THE PATCH ‼️** prereqs: don't be stupid and have fast fingers sign in as existing user on network not available page
> 
> # Steps:
> 1. Sign out and turn off your wifi
> 2. Open kiosk app of choice and wait for the network page to show
> 3. DO NEXT STEPS AS FAST AS POSSIBLE
> 4. Click add network and quickly type in random letters and hit enter
> 5. Quickly press diagnose, wifi, open in settings, and then press Close at the bottom right to exit out back to the wifi page
> 6. As fast as you can press log in as existing and sign in
> 7. Close the school window in the top right and there should be an unblocked page behind it
> 
> If all went well you should have an unblocked SKIOVOX container.

this does work but it needs 124 specifically because the multi user sign in popup doesn't show up on this version

**Reply by FunnyCorgi** [reply]
Thanks! Even though I have to wait for my school to update us to 124.
